doc/tutorials/using_vxworks_as_uos.rst

```diff
@@ -5,7 +5,7 @@ Run VxWorks as the User VM
 
 `VxWorks`_\* is a real-time proprietary OS designed for use in embedded systems requiring real-time, deterministic
 performance. This tutorial describes how to run VxWorks as the User VM on the ACRN hypervisor
-based on Clear Linux 29970 (ACRN tag v1.1).
+based on Ubuntu Service VM (ACRN tag v2.0).
```

Comment:
Just double-checking here... is it really **v2.0** that we used for the last validation, and not **v2.2**?

Reply:
Yes, there is one known limitation that Vxworks can't boot up since following PR is merged into v2.1:
https://github.com/projectacrn/acrn-hypervisor/pull/4692

Reply:
Thanks for the confirmation @fuzhongl 

Is this problem tracked (kind of implicitly) in either of these issues: https://github.com/projectacrn/acrn-hypervisor/issues?q=is%3Aissue+is%3Aopen+vxworks?

Reply:
Yes, it is tracked by https://github.com/projectacrn/acrn-hypervisor/issues/5151